src/mtx/commands/uplink/getEvents.ts

```diff
@@ -0,0 +1,174 @@
+/**
+ * Uplink command to get device events.
```

Comment:
it should probably match the downlink command description

Reply:
fixed